### PR TITLE
Add trace visualization as a FlameGraph

### LINKED ISSUE #525: Support trace view as a flame graph
I am implementing jaeger, it really looks nice.
But I got a problem.
I want to collect spans on one line in tracing view, but I cannot find way.

[As-IS]
image

[To-Be]
image

Maybe I can't find the doc... could you help ?

### PR DESCRIPTION
**Pull request by pavelpashkovsky**
## Which problem is this PR solving?
We added the ability to visualize a trace as a flamegraph and thought it might be worth contributing upstream! Would love feedback on if it resolves this issue: https://github.com/jaegertracing/jaeger-ui/issues/390

Resolves #525

## Short description of the changes

https://user-images.githubusercontent.com/7372044/183039257-26fd8cee-1d57-411a-a31c-032a2d9a0948.mp4

### CODE REVIEW

**Review by yurishkuro — COMMENTED** [review]
Thanks for contributing! 🥳 

A couple of questions on the view:
  1. The table on the left seems sorted by Self column - is it possible to indicate that visually?  I can't tell if the sorting is by Self or Total column, can only infer it from the data
  2. There is a Total/Self cell in the table that shows "< 0.01 sec" - is it informative? As I understand, "total" here refers to an implied span outside of the actual spans (judging by the flame graph itself, the top purple row), so under what condition could Total/Self be non-zero?

Lastly, please make sure all commits in the PR are signed (it's a CNCF licensing requirement).

File: packages/jaeger-ui/src/components/TracePage/TraceFlamegraph/index.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  const convertedProfile = convertJaegerTraceToProfile(trace.data);
+
+  return (
+    <div className="Flamegraph-wrapper">
+      <FlamegraphRenderer colorMode="light" profile={convertedProfile} />
+    </div>
+  );
+};
+
+export default TraceFlamegraph;
```
Comment (nofar9792) [inline packages/jaeger-ui/src/components/TracePage/TraceFlamegraph/index.tsx:30]: maybe add some basic tests like "does not explode" and snapshot?

**Review by yurishkuro — DISMISSED** [review]

**Comment by yurishkuro** [comment]
Thank you!

**Comment by bobrik** [comment]
There are some kinks with this: #1012.

**Pull request merged**